Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +17,114 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            var age = Console.ReadLine();
```

Comment:
There is a pattern for prompting something. You could make a method for those 2 lines:

```cs
public static string Prompt(string message)
{
                Console.WriteLine(message);
                return Console.ReadLine();
} 
```

Reply:
You can also include a null/empty string validation